changelog

```diff
@@ -1,6 +1,9 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 V.Next
 ----------
+
+Version 4.0.2
+----------
```

Comment:
Can you add the changelog entry for updating common version.
- [PATCH] Update common @7.0.0